bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```
@@ -1327,6 +1367,27 @@ public void onNewMessageEvent(EventObject event) {
             default:
                 break;
         }
+        int preset = 0;
+        if (key.startsWith(KEY_FM_PRESET)) {
+            try {
+                preset = Integer.parseInt(key.substring(KEY_FM_PRESET.length()));
+            } catch (NumberFormatException e) {
+            }
+            key = KEY_FM_PRESET;
+        } else if (key.startsWith(KEY_DAB_PRESET)) {
+            try {
+                preset = Integer.parseInt(key.substring(KEY_DAB_PRESET.length()));
+            } catch (NumberFormatException e) {
+            }
+            key = KEY_DAB_PRESET;
+        } else if (key.startsWith(KEY_IRADIO_PRESET)) {
+            try {
+                preset = Integer.parseInt(key.substring(KEY_IRADIO_PRESET.length()));
+            } catch (NumberFormatException e) {
+            }
```

Consider adding a comment here why this exception is ignored.

---

Done